apps/meteor/app/lib/server/lib/getMemSettings.ts

```diff
@@ -0,0 +1,4 @@
+import { Settings } from '@rocket.chat/models';
+import mem from 'mem';
+
+export const getSettingCached = mem(Settings.getValueById, { maxAge: 10000 });
```

Comment:
needed?

Reply:
Yeah. But since you can remove the usage of this on loadMessageHistory, you can remove the file and keep the declaration on messages as it was